modules/i3s/src/lib/parsers/parse-slpk/parse-slpk.ts

```diff
@@ -1,9 +1,13 @@
 import {SLPKLoaderOptions} from 'modules/i3s/src/i3s-slpk-loader';
-import {parseZipCDFileHeader} from '../parce-zip/cd-file-header';
-import {parseZipLocalFileHeader} from '../parce-zip/local-file-header';
+import {parseZipCDFileHeader} from '../parse-zip/cd-file-header';
+import {parseZipLocalFileHeader} from '../parse-zip/local-file-header';
 import {SLPKArchive} from './slpk-archieve';
+
 /**
  * Returns one byte from the provided buffer at the provided position
+ * @param {number} offset - position where to read
```

Comment:
Nit. In TS it is not required to add types in TSDoc because they are in the code